### PR TITLE
spdlog@1.15.0

### PR DESCRIPTION
**Pull request by c8ef**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @Vertexwahn, modules you maintain (spdlog) have been updated in this PR. Please review the changes.

File: modules/spdlog/1.15.0/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module(
+    name = "spdlog",
+    version = "1.15.0",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "fmt", version = "11.0.2")
+bazel_dep(name = "rules_cc", version = "0.0.16")
```
Comment (Vertexwahn) [inline modules/spdlog/1.15.0/MODULE.bazel:8]: Should we use 0.1.0 here?

File: modules/spdlog/1.15.0/presubmit.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+    - debian10
+    - debian11
+    - macos
+    - macos_arm64
+    - ubuntu2004
+    - ubuntu2204
```
Comment (Vertexwahn) [inline modules/spdlog/1.15.0/presubmit.yml:8]: Add ubuntu2404

**Review by Vertexwahn — APPROVED** [review]

**Review by bazel-io — APPROVED** [review]
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**